apps/condo/jest.config.js

```diff
@@ -45,7 +45,10 @@ module.exports = {
             // NOTE: need to pass uuid export syntax through babel
             transformIgnorePatterns: ['/node_modules/(?!(uuid|nanoid|msgpackr|n2words)/)'],
             moduleNameMapper: {
-                '@open-condo/(.*)': '<rootDir>/../../packages/$1',
+                '^@open-condo/(.*?)/(.*)$': [
+                    '<rootDir>/../../packages/$1/$2',
+                    '<rootDir>/../../packages/$1/dist/$2', // ts package has transpiled modules at dist
```

Comment:
Why do you need that?

The line above was for `@open-condo/next` package, where files are not converted to js. Your lib is transpiled correctly, so this config is wrong.

Maybe we can change it instead to transpile not all `@open-condo` packages, but only `next` instead?

Reply:
This is the only way I've found to make this jest-project work. The `exports` field within package.json is not working here. Try to run `apps/condo/domains/notification/helpers/sendRemoteClientsUpgradeAppNotifications.spec.js`.
<img width="640" alt="image" src="https://github.com/user-attachments/assets/28d2ca8c-4121-46bc-aec6-bed7bafa9aca" />
I can write my own rule for `@open-condo/billing` in this jest-project.

Reply:
Trying to change to `'@open-condo/next/(.*)': '<rootDir>/../../packages/next/$1',`